src/providers/Watcher/index.tsx

```diff
@@ -64,6 +67,31 @@ const Watcher = () => {
     }
   };
 
+  const sendNotification = ({
+    notificationPayload,
+    toastPayload,
+  }: {
+    // notification payload without id
+    notificationPayload: Omit<Parameters<typeof appActions.addNotification>[0], 'id'>;
+    toastPayload?: { message: string; type?: ToastOptions['type'] };
+  }) => {
+    const notificationId = nanoid();
```

Comment:
Do we have a redux id already? When you were off @mjadach-iv talked about `nanoid` but I can't quite remember it

Reply:
had to move it out of redux because the notification toast also needed to get the id to know what notification was clicked. That is why it is required in the payload now.

Reply:
You get id from redux toolkit on every action, check the messages async actions. If you can't you the same logic for getting id, we can stay with this one.

Reply:
It's a good implementation